addons/gloot/ctrl_inventory_item_rect.gd

```diff
@@ -75,8 +76,13 @@ func _get_item_position() -> Vector2:
 
 func _ready() -> void:
     if item && ctrl_inventory:
+        focus_mode = Control.FOCUS_ALL

```

Comment:
A short comment on why we're focusing would be useful. Otherwise, someone might consider this part unnecessary and remove it.